doc/reference/encoding_a_column.rst

```diff
@@ -12,6 +12,7 @@ Encoding a column
    GapEncoder
    MinHashEncoder
    SimilarityEncoder
+   SentenceEncoder
```

Comment:
I wonder about the name. I get that it reflects the name of the underlying library, but for a user who isn't familiar with it or with this implementation detail it might not be obvious what this does. maybe LanguageEncoder, TextEmbedding, ...

Reply:
TextEmbedding is nice, the only issue is vectors and embeddings are close concepts, so people might not understand its difference with other string-based encoders like MinHash. What about LLMEncoder?

Reply:
I would go for "TextEncoder".

A detail, but in a list like the above, I would like it to be in a separate sub(sub)section: "Deep Learning", with a tiny text in the beginning that explains the need to install other libraries and points to the part of the documentation where we explain how to do this.